src/renderer/views/AppSettings/subcomponents/DefaultDirSelect.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button, InputLabel, TextField } from '@mui/material';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+const { dialog } = require('electron').remote;
+
+interface Props {
+  defaultDirPath: string;
+  onChange: (newDirPath: string) => void;
+}
+
+export default function DefaultDirSelect({ defaultDirPath, onChange }: Props) {
+  const { t } = useTranslation();
+  return (
+    <>
+      <InputLabel id='dir-select-label'>
+        {t('AppSettings.defaultDirectory.description')}
+      </InputLabel>
+      <TextField label='dir-select-label' value={defaultDirPath} />
```

Comment:
I know this is mocked directory picker anyway but maybe this can be a good tip. 

The label here just displays the given string as the MUI label:

![image](https://user-images.githubusercontent.com/34404855/141748786-e5624ce5-cec9-4388-adb7-50b464bf0d0d.png)

![image](https://user-images.githubusercontent.com/34404855/141748813-ef58faf5-4201-4cca-9df0-9ad96409630a.png)

If you want to say, which element labels the field you can use the `aria-labelledby` attribute (see more [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute))

Reply:
Huh, I think I meant to use `labelId` as with `Select`. A mistake anyway, since the property does not exist on `TextField`s. Just to be clear, do you want me to merge anyway and just make sure it will be correct on the final UI implementation?

Reply:
@LostInStatic yeah you can merge no bother 👍